app/interactions/games/create_neutral_force_tokens.rb

```diff
@@ -3,16 +3,22 @@ class CreateNeutralForceTokens < ActiveInteraction::Base
     object :game
 
     def execute
-      number_of_houses = game.houses.count
-      neutral_force_tokens_setup.map do |territory, range_of_houses|
-        next if range_of_houses.flatten.exclude?(number_of_houses)
-        create_neutral_force(range_of_houses, territory)
+      neutral_force_tokens_setup.map do |territory, sides|
+        token = select_token_side(sides)
+        next if token.blank?
+        create_neutral_force(token, territory)
       end
       game.neutral_force_tokens
     end
 
     private
 
+    def select_token_side(sides)
+      sides.values.find do |side|
+        Array(side["range"]).include?(game.houses.count)
```

Comment:
`[side["range"]]` is significantly faster than `Array(side["range"])`

Reply:
got it, I was trying to avoid to call flatten, maybe to avoid all that we could always define range an array in YAML, what about that?